src/main/java/gov/cms/mat/cql_elm_translation/utils/HumanReadableUtil.java

```diff
@@ -58,14 +58,24 @@ public static List<String> getMeasureTypes(Measure measure) {
   }
 
   public static String getStratification(Measure measure) {
+    // Collects and returns all stratification descriptions for display
     if (CollectionUtils.isNotEmpty(measure.getGroups())) {
+      String allDescriptions = "";
```

Comment:
Strings in JAVA are immutable, so every time we modify the original string it creates a new copy, which is unnecessary in our case. May be we could use StringBuilder ? StringBuilder also helps us with safely append a sub-string.